shared/profile/render.shared.js

```diff
@@ -14,3 +18,14 @@ export function folderIconProps (folder, style = {}) {
     style: {...style, color},
   }
 }
+
+export function missingProofs (userProofs: Array<Proof>, onClick: (proof: MissingProof) => void): Array<MissingProof> {
+  const availableProofs = ['btc'].concat(_.without(_.keys(proveCommon.ProofType), 'none', 'keybase', 'dns'))
+    .map(type => ({type, message: proveMessage(type), onClick}))
+  const userProofTypes = userProofs.map(p => p.type)
+  const missingProofs = availableProofs.filter(ap => userProofTypes.indexOf(ap.type) === -1)
```

Comment:
Alternative: `_.difference` `availableProofs` and `userProofTypes` and remove website types if applicable first and then map to `MissingProof` objects